### PR TITLE
utils: utilities for managing errbot

### PR DESCRIPTION
**Pull request by meetmangukiya**
# Reviewers Checklist

- [ ] Appropriate logging is done.
- [ ] Appropriate error responses.
- [ ] Handle every possible exception.
- [ ] Make sure there is a docstring in the command functions. Hint: Lookout for
  `botcmd` and `re_botcmd` decorators.
- [ ] See that 100% coverage is there.
- [ ] See to it that mocking is not done where it is not necessary.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -10,6 +10,7 @@ RUN apk add --no-cache libffi openssl \
            libc-dev \
            libffi-dev \
            openssl-dev \
+    && apk add --no-cache git \
```
Comment (yukiisbored) [inline Dockerfile]: Just add git to line 7

Comment (meetmangukiya) [inline Dockerfile]: done :+1:

File: utils/utils.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+class Utils(BotPlugin):
+    """
+    Some random utilities
+    """
+
+    @botcmd
+    def sync(self, msg, arg):
+        """Sync the repository from github."""  # Ignore QuotesBear
+        os.chdir(os.environ.get('COBOT_ROOT'))
+        initial_head = run('git log --oneline -1')
```
Comment (yukiisbored) [inline utils/utils.py]: Can you find a Python git library? I really don't like going to shell

Comment (yukiisbored) [inline utils/utils.py]: * http://www.pygit2.org/
* https://github.com/gitpython-developers/GitPython

Comment (meetmangukiya) [inline utils/utils.py]: there was `GitPython`, but it didn't have `rebase` :confused:

Comment (meetmangukiya) [inline utils/utils.py]: Done :+1:

**Comment by yukiisbored** [comment]
ack https://github.com/coala/corobo/pull/136/commits/fc9ccc8bfebfaa8731a3738e6891d65ea0c2a647

**Comment by yukiisbored** [comment]
ack https://github.com/coala/corobo/pull/136/commits/337dd98973d2ee0fc232292bc55885f29d2ae0b6 https://github.com/coala/corobo/pull/136/commits/d015fbf66cf47a54b260befe6f2d6efcc474521e

**Comment by yukiisbored** [comment]
Wait, i just noticed something

**Comment by yukiisbored** [comment]
unack https://github.com/coala/corobo/pull/136/commits/fc9ccc8bfebfaa8731a3738e6891d65ea0c2a647https://github.com/coala/corobo/pull/136/commits/337dd98973d2ee0fc232292bc55885f29d2ae0b6 https://github.com/coala/corobo/pull/136/commits/d015fbf66cf47a54b260befe6f2d6efcc474521e

**Comment by yukiisbored** [comment]
@meetmangukiya Can you squash the commit? I think you don't need 3 of them just for a single feature added. Also, please add the git library to the requirements so it'll be installed.

**Comment by meetmangukiya** [comment]
reack  3747904b40f0df371ab21daf140ed1ed825dff5e feb9a633f3e915a702309c1a6cc8e19814a203f3

**Comment by meetmangukiya** [comment]
@yukiisbored rebuild a container after this merge, then we'll never need to rebuild :tada:

**Comment by meetmangukiya** [comment]
@rultor merge

**Pull request merged**